test/models/ToolpadHome.ts

```diff
@@ -40,7 +40,7 @@ export class ToolpadHome {
     });
     this.newAppNameInput = this.newAppDialog.locator('label:has-text("name")');
     this.newAppTemplateSelect = this.newAppDialog.locator(
-      '[aria-haspopup="listbox"]:has-text("blank")',
+      '[aria-haspopup="listbox"]:has-text("demo")',
```

Comment:
I know this wasn't correct before the PR already, but we should try to select this with the `Use template` label if there's a way.

Reply:
How would we select the option with the `Use template` label? (if I'm understanding what you're saying correctly)

Reply:
https://github.com/mui/mui-toolpad/blob/28a36db057e184243941db39d7a1ef45130aaa76/test/integration/appTemplates.spec.ts#L33-L35

Reply:
> How would we select the option with the Use template label?
It's not the label for one of the options, it's the label for the whole select input.

Reply:
I've done it with the method @Janpot mentioned up there, seems to work